provider/pkg/schema/gen.go

```diff
@@ -595,7 +600,7 @@ func GatherPackage(supportedResourceTypes []string, jsonSchemas []jsschema.Schem
 		},
 	}
 
-	return &p, &metadata, nil
+	return &p, &metadata, reports, nil
 }
 
 // context holds shared information for a single CF JSON schema.
```

Comment:
Not part of this PR but `context` is a confusing type name given the prevalence of `context.Context`

Reply:
Yes I agree 😑